### DISCUSSION TITLE
Error Handling POST to Rest API

### DISCUSSION THREAD

**Discussion by sjarifHD** [Budibase/budibase#4940] [Help]
I have an budibase apps which is connected to datasource REST. I create form for submitting data to REST, I can successfully integrate that, but how about handling error message from REST?

Currently action from button is like this
![image](https://user-images.githubusercontent.com/9961361/158948890-ec9963b5-9f3a-4e27-9931-7a65de94691e.png)

**Comment by mjashanks** [comment]
Unortunately this is not possible at the moment. However, within the next month  we will be adding a "Continue If.." step, whereby you can halt the actions executions under certain conditions - i.e. a failed request.
